src/io/file_descriptor.cr

```diff
@@ -177,63 +145,14 @@ class IO::FileDescriptor < IO
     pp.text inspect
   end
 
-  private def unbuffered_read(slice : Bytes)
-    read_syscall_helper(slice, "Error reading file") do
-      # `to_i32` is acceptable because `Slice#size` is a Int32
-      LibC.read(@fd, slice, slice.size).to_i32
-    end
-  end
-
-  private def unbuffered_write(slice : Bytes)
-    write_syscall_helper(slice, "Error writing file") do |slice|
-      LibC.write(@fd, slice, slice.size).tap do |return_code|
-        if return_code == -1 && Errno.value == Errno::EBADF
-          raise IO::Error.new "File not open for writing"
-        end
-      end
-    end
-  end
-
-  private def add_read_event(timeout = @read_timeout)
-    event = @read_event ||= Scheduler.create_fd_read_event(self)
-    event.add timeout
-    nil
-  end
-
-  private def add_write_event(timeout = @write_timeout)
-    event = @write_event ||= Scheduler.create_fd_write_event(self)
-    event.add timeout
-    nil
-  end
-
   private def unbuffered_rewind
-    seek(0, IO::Seek::Set)
-    self
+    self.pos = 0
```

Comment:
Why the change?

Reply:
`pos=(value)` calls `seek(value)` and `Seek::Set` is default. So `seek(0)` would be the most direct call here.

Reply:
No reason for the change, it's trivial to see it's the same but I can revert it if you want.